enterprise/models.py

```diff
@@ -2554,6 +2554,25 @@ class SystemWideEnterpriseUserRoleAssignment(EnterpriseRoleAssignmentContextMixi
 
     role_class = SystemWideEnterpriseRole
 
+    enterprise_customer = models.ForeignKey(
```

Comment:
Should we be making this be required now or is it too early to do that?

Reply:
It's too early, and there will be some rows for which this column must be null (i.e. with the `enterprise_openedx_operator` role).  A PR some time after this one will put in some validation logic around this field and `applies_to_all`, though.